conda_requirements.yml

```diff
@@ -1,56 +1,56 @@
-name: nwtenv
+name: NWTOPT
 channels:
-- conda-forge
-- defaults
+  - conda-forge
+  - defaults
 dependencies:
-- _libgcc_mutex=0.1
-- _openmp_mutex=4.5
-- blas=1.0
-- bottleneck=1.3.2
-- ca-certificates=2021.7.5
-- certifi=2021.5.30
-- cloudpickle=1.6.0
-- colorama=0.4.4
-- conda-pack
-- decorator=5.0.9
-- future=0.18.2
-- hyperopt=0.2.5
-- ld_impl_linux-64=2.35.1
-- libblas=3.9.0
-- libcblas=3.9.0
-- libffi=3.3
-- libgcc-ng=9.3.0
-- libgfortran-ng=11.1.0
-- libgfortran5=11.1.0
-- libgomp=9.3.0
-- liblapack=3.9.0
-- libopenblas=0.3.17
-- libstdcxx-ng=9.3.0
-- ncurses=6.2
-- networkx=2.3
-- numexpr=2.7.3
-- numpy=1.21.1
-- openssl=1.1.1k
-- pandas=1.3.0
-- pip=21.2.2
-- pymongo=3.12.0
-- python=3.8.11
-- python-dateutil=2.8.2
-- python_abi=3.8
-- pytz=2021.1
-- readline=8.1
-- scipy=1.7.0
-- setuptools=52.0.0
-- six=1.16.0
-- sqlite=3.36.0
-- tk=8.6.10
-- tqdm=4.62.0
-- wheel=0.36.2
-- xz=5.2.5
-- zlib=1.2.11
-- pip
-- pathlib=1.0
-- dill=0.3.4
-- wrapt=1.12.1
-- multiprocess=0.70.12.2
-
+  - _libgcc_mutex=0.1=main
```

Comment:
this level of specificity will not be platform agnostic. Might want to simplify it a bit (e.g. just the higher level packages and let conda grab the deps)